Configuration/FlexForms/ServiceBw2FlexFormConfiguration.xml

```diff
@@ -0,0 +1,150 @@
+                                <internal_type>db</internal_type>
+                                <allowed>pages</allowed>
+                                <size>1</size>
+                            </config>
+                        </TCEforms>
+                    </settings.organisationseinheiten.pidOfListPage>
+                    <settings.leistungen.pidOfListPage>
+                        <TCEforms>
+                            <label>LLL:EXT:service_bw2/Resources/Private/Language/FlexForms.xlf:leistungen</label>
+                            <config>
+                                <type>group</type>
+                                <internal_type>db</internal_type>
+                                <allowed>pages</allowed>
+                                <size>1</size>
+                            </config>
+                        </TCEforms>
+                    </settings.leistungen.pidOfListPage>
+                    <settings.lebenslagen.pidOfListPage>
+                        <TCEforms>
+                            <label>LLL:EXT:service_bw2/Resources/Private/Language/FlexForms.xlf:lebenslagen</label>
+                            <config>
+                                <type>group</type>
+                                <internal_type>db</internal_type>
+                                <allowed>pages</allowed>
+                                <size>1</size>
+                            </config>
+                        </TCEforms>
+                    </settings.lebenslagen.pidOfListPage>
+                </el>
+            </ROOT>
+        </sPagesList>
+        <sPagesDetail>
+            <ROOT>
+                <TCEforms>
+                    <sheetTitle>LLL:EXT:service_bw2/Resources/Private/Language/FlexForms.xlf:sheet_detail_pages
+                    </sheetTitle>
+                </TCEforms>
+                <type>array</type>
+                <el>
+                    <settings.organisationseinheiten.pidOfDetailPage>
+                        <TCEforms>
+                            <label>LLL:EXT:service_bw2/Resources/Private/Language/FlexForms.xlf:organisationseinheiten
+                            </label>
+                            <config>
+                                <type>group</type>
+                                <internal_type>db</internal_type>
+                                <allowed>pages</allowed>
+                                <size>1</size>
+                            </config>
+                        </TCEforms>
+                    </settings.organisationseinheiten.pidOfDetailPage>
+                    <settings.leistungen.pidOfDetailPage>
+                        <TCEforms>
+                            <label>LLL:EXT:service_bw2/Resources/Private/Language/FlexForms.xlf:leistungen</label>
+                            <config>
+                                <type>group</type>
+                                <internal_type>db</internal_type>
```

Comment:
internal_type is deprecated with TYPO3 12.0:
Deprecation: #96983 - TCA internal_type